### PR TITLE
Gophercon US Challenge Series Blog Post

### PR DESCRIPTION
**Pull request by deelawn**
This blog post explains a bit about what we did for the challenge series. It then goes on to present the challenges as they were presented at gophercon, including annotated realm code, solutions, and explanations.

### CODE REVIEW

**Review by thehowl — APPROVED** [review]
LGTM

File: posts/2024-07-29_gc24-challenge-series/README.md [hunk 1]
```diff
@@ -0,0 +1,922 @@
+---
+publication_date: 2024-07-29T00:00:00Z
+slug: gc24-challenge-series
+tags: [gnoland, gophercon, gc24, challenge-series]
+authors: [deelawn]
+---
+
+# Gophercon US: gno.land's Challenge Series Contributions
+
+gno.land is pleased to have been granted the opportunity to provide a series of challenges for the Challenge Series at the 2024 Gophercon in Chicago. We enjoyed writing them and hope the participants had a good time and found it interesting to learn about gno.land and blockchains in general.
```
Comment (nprimmer) [inline posts/2024-07-29_gc24-challenge-series/README.md:10]:
```suggestion
gno.land is pleased to have been granted the opportunity to provide a series of challenges for the Challenge Series at the 2024 GopherCon in Chicago. We enjoyed writing them and hope the participants had a good time and found it interesting to learn about gno.land and blockchains in general.
```

File: posts/2024-07-29_gc24-challenge-series/README.md [hunk 2]
```diff
@@ -0,0 +1,922 @@
+---
+publication_date: 2024-07-29T00:00:00Z
+slug: gc24-challenge-series
+tags: [gnoland, gophercon, gc24, challenge-series]
+authors: [deelawn]
+---
+
+# Gophercon US: gno.land's Challenge Series Contributions
```
Comment (nprimmer) [inline posts/2024-07-29_gc24-challenge-series/README.md:8]:
```suggestion
# GopherCon US: gno.land's Challenge Series Contributions
```

File: posts/2024-07-29_gc24-challenge-series/README.md [hunk 3]
```diff
@@ -0,0 +1,922 @@
...
+	newValue += mask[time.Now().Unix()%int64(len(mask))]
+	newValue /= 2
+	newValueStr := time.Unix(newValue, 0).Format("2006-01-02 15:04:05")
+	p3.UnmaskMeIfYouWant(newValueStr, ^(newValue ^ 0xdeadbeef))
+}
+```
+
+## Final Words
+
+We enjoyed coming up with these challenges and were happy to contribute to the Gophercon Challenge Series -- from coming up with the challenges, theming them, locking down certain gno.land features, and setting up infrastructure -- a lot of work went into this. We hope all participants were able to learn a bit more about gno.land and had fun doing it. Hopefully we'll be back next year 😁
```
Comment (nprimmer) [inline posts/2024-07-29_gc24-challenge-series/README.md:922]:
```suggestion
We enjoyed coming up with these challenges and were happy to contribute to the GopherCon Challenge Series -- from coming up with the challenges, theming them, locking down certain gno.land features, and setting up infrastructure -- a lot of work went into this. We hope all participants were able to learn a bit more about gno.land and had fun doing it. Hopefully we'll be back next year 😁
```

**Comment by deelawn** [comment]
🎉 thanks Leon

**Pull request merged**